storybook/pages/CommunitiesPortalLayoutPage.qml

```diff
@@ -25,7 +25,11 @@ SplitView {
                 readonly property string locale: ""
                 readonly property int unreadNotificationsCount: 42
                 readonly property string communityTags:
-                    JSON.stringify({"Activism":"✊","Art":"🎨","Blockchain":"🔗","Books & blogs":"📚","Career":"💼"})
+                    JSON.stringify({"Activism":"✊","Art":"🎨","Blockchain":"🔗","Books & blogs":"📚","Career":"💼","Collaboration":"🤝","Commerce":"🛒","Culture":"🎎","DAO":"🚀","DIY":"🔨","DeFi":"📈",
```

Comment:
@osmaczko does it make sense to have it in a store or maybe directly in the backend instead?